### PR TITLE
fix test case for ubuntu docker

### PR DESCRIPTION
**Pull request by bright-starry-sky**
Temporary fix test case for ubuntu docker. The mv command file is in the /bin directory on ubuntu docker. 
I will add a renameDir method to FileUtils at later.

### CODE REVIEW

File: src/kvstore/test/NebulaStoreTest.cpp [hunk 1]
```diff
@@ -760,13 +760,13 @@ TEST(NebulaStoreTest, ThreeCopiesCheckpointTest) {
         fs::FileUtils::remove(folly::stringPrintf("%s/data", rm.data()).c_str(), true);
         fs::FileUtils::remove(folly::stringPrintf("%s/wal", rm.data()).c_str(), true);
         std::string mv = folly::stringPrintf(
-                "/usr/bin/mv %s/disk%d/nebula/0/checkpoints/snapshot/data %s/disk%d/nebula/0/data",
+                "mv %s/disk%d/nebula/0/checkpoints/snapshot/data %s/disk%d/nebula/0/data",
                 rootPath.path(), i , rootPath.path(), i);
         sleep(1);
         auto ret = system(mv.c_str());
```
Comment (Shylock-Hg) [inline src/kvstore/test/NebulaStoreTest.cpp]: Consider the function `rename` in stdio.h?

Comment (bright-starry-sky) [reply src/kvstore/test/NebulaStoreTest.cpp]: `std::rename` is for files only, not directories.
Actually, I'm more optimistic with `std::filesystem::rename` . However, the current environment does not yet allow for the use of `std::filesystem::rename`, which will take some time.

Comment (Shylock-Hg) [reply src/kvstore/test/NebulaStoreTest.cpp]: The `rename` could apply to folder according to the man page. And I tried in nebula-dev containers.

Comment (bright-starry-sky) [reply src/kvstore/test/NebulaStoreTest.cpp]: > The `rename` could apply to folder according to the man page. And I tried in nebula-dev containers.

Aha，Thanks for your verification ! Would you like to provide the PR？

Comment (Shylock-Hg) [reply src/kvstore/test/NebulaStoreTest.cpp]: > > The `rename` could apply to folder according to the man page. And I tried in nebula-dev containers.
> 
> Aha，Thanks for your verification ! Would you like to provide the PR？

Ok.

Comment (bright-starry-sky) [reply src/kvstore/test/NebulaStoreTest.cpp]: Close this one. And PR #1387 will  fix this bug.

**Pull request closed without merging**